sdk/servicefabric/Azure.ResourceManager.ServiceFabric/api/Azure.ResourceManager.ServiceFabric.netstandard2.0.cs

```diff
@@ -54,135 +54,136 @@ public partial class ApplicationResourceData : Azure.ResourceManager.Models.Trac
         public string TypeVersion { get { throw null; } set { } }
         public Azure.ResourceManager.ServiceFabric.Models.ApplicationUpgradePolicy UpgradePolicy { get { throw null; } set { } }
     }
-    public partial class ApplicationTypeResource : Azure.ResourceManager.ArmResource
+    public partial class ServiceFabricApplicationTypeResource : Azure.ResourceManager.ArmResource
```

Comment:
It is weird that a "SomethingType" is a resource. would `TypeProviderResource` make it better?

Reply:
It's not `TypeProvider` but the type name of the application, so I think here `AppicationType` is ok